crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -2139,7 +2139,22 @@ fn has_cjs_export(p: &Program) -> bool {
 
     if let Program::Module(m) = p {
         // Check for imports/exports
-        if m.body.iter().any(ModuleItem::is_module_decl) {
+        if m.body.iter().any(|item| {
+            item.as_module_decl().map_or(false, |module_decl| {
+                module_decl
+                    .as_import()
+                    .and_then(|import| import.asserts.as_ref())
+                    .map_or(true, |asserts| {
+                        asserts.props.iter().any(|assert| {
+                            assert
+                                .as_prop()
+                                .and_then(|prop| prop.as_key_value())
+                                .and_then(|kv| kv.key.as_ident())
+                                .map_or(true, |ident| &*ident.sym != "turbopackHelper")
+                        })
+                    })
```

Comment:
This should be its own function `is_turbopack_helper_module_decl(module_decl) -> bool`.